schemas/dependencies/endpoint-monitoring-provider-1.yml

```diff
@@ -0,0 +1,79 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /dependencies/endpoint-monitoring-provider-1.yml
+  labels:
+    "$ref": "/common-1.json#/definitions/labels"
+  name:
+    type: string
+  description:
+    type: string
+  provider:
+    type: string
+    enum:
+    - blackbox-exporter
+    description: The provider defines which monitoring implementation will be used for endpoints
+  timeout:
+    type: string
+    description: Timeout for the monitoring check, e.g. 10s
```

Comment:
Should this have a regex so that it is clearer what it is accepted, e.g. `^\d+[smhd]$`?

Reply:
fixed in c209757